### PR TITLE
fix/approve payments api docs

### PR DESCRIPTION
**Pull request by caiocampoos**
Screenshot 2023-08-07 at 14 31 18

### CODE REVIEW

File: src/swaggers/openpix.yml [hunk 1]
```diff
@@ -1207,6 +1214,17 @@ paths:
                     comment: payment comment
                     correlationID: payment1
                     sourceAccountId: my-source-account-id
+                  transaction:
+                    value: 100
+                    endToEndId: transaction-end-to-end-id
+                    time: 2023-03-20T13:14:17.000Z
+                  destination:
+                    name: Dan
```
Comment (Eckzzo) [inline src/swaggers/openpix.yml:1222]: Is this correct? shouldn't it be a string?

File: src/swaggers/openpix.yml [hunk 2]
```diff
@@ -1207,6 +1214,17 @@ paths:
...
+                    taxID: '31324227036'
+                    pixKey: c4249323-b4ca-43f2-8139-8232aab09b93
+                    bank: A Bank
```
Comment (Eckzzo) [inline src/swaggers/openpix.yml:1225]: Is this correct? shouldn't it be a string?

File: src/swaggers/openpix.yml [hunk 3]
```diff
@@ -1207,6 +1214,17 @@ paths:
                     comment: payment comment
                     correlationID: payment1
                     sourceAccountId: my-source-account-id
+                  transaction:
+                    value: 100
+                    endToEndId: transaction-end-to-end-id
+                    time: 2023-03-20T13:14:17.000Z
```
Comment (Eckzzo) [inline src/swaggers/openpix.yml:1220]: Is this correct? shouldn't it be a string?

**Review by Eckzzo — APPROVED** [review]

File: src/swaggers/openpix.yml (hunk 3 above)
Comment (caiocampoos) [reply src/swaggers/openpix.yml:1220]: it should, idk why generated like that

Comment (caiocampoos) [reply src/swaggers/openpix.yml:1220]: yml just works that way, on the docs is right

File: src/swaggers/openpix.yml [hunk 4]
```diff
@@ -1191,14 +1191,21 @@ paths:
               correlationID: payment1
       responses:
         '200':
-          description: Payment destination account information
+          description: The payment approved
```
Comment (Eckzzo) [inline src/swaggers/openpix.yml:1194]: The approved payment

File: src/swaggers/openpix.yml (hunk 3 above)
Comment (Eckzzo) [reply src/swaggers/openpix.yml:1220]: I would rather keep a pattern, everything string or not string, I don't like the mixing tbh

**Pull request merged**